modules/aws-http4s/src/smithy4s/aws/AwsCredentialsProvider.scala

```diff
@@ -62,6 +62,8 @@ class AwsCredentialsProvider[F[_]](implicit F: Temporal[F]) {
     Resource
       .eval(fromEnv)
       .map(F.pure)
+      // TODO: Ensure minimal delay when these endpoints don't exist, e.g.
+      // when running locally.
```

Comment:
Didn't feel it was appropriate to make any more changes to the provider as part of this PR, so just leaving this here.

Reply:
Fair nuff